.github/workflows/build.yml

```diff
@@ -85,7 +85,7 @@ jobs:
 
   test-integration:
     name: Integration Tests
-    runs-on: windows-2019
```

Comment:
Should we just move to testing both platforms?

Reply:
Sure, but another PR preferred.